fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexer.java

```diff
@@ -525,6 +535,22 @@ public void close() {
         }
     }
 
+    private void maybeLogBuildProgress(Subspace subspace, Tuple startTuple, Tuple endTuple, Tuple realEnd) {
+        if (LOGGER.isInfoEnabled()
+            && (progressLogIntervalMillis > 0
```

Comment:
Shouldn't this be `>= 0` if you want to log for each commit?

Reply:
it does an `|| progressLogIntervalMillis == 0`, but it would be better to just do `>=` on both

Reply:
In response to your comment I changed it to be simpler, but `System.currentTimeMillis` can go backwards if the system clock changes (e.g. in response to NTP). While I'm not too worried about it changing and logging a little extra, or a little less if you have it set to every second, I think dropping logs when it is set to log always would not be great.

Also, System.nanoTime is only monotonic some of the time: https://bugs.java.com/bugdatabase/view_bug.do?bug_id=6458294